snowplow-tracker/src/main/java/com/snowplowanalytics/core/tracker/Subject.kt

```diff
@@ -179,13 +179,28 @@ class Subject(context: Context, config: SubjectConfigurationInterface?) {
             field = depth
             standardPairs[Parameters.COLOR_DEPTH] = depth.toString()
         }
+
+    var useContextResourcesScreenResolution: Boolean? = false
```

Comment:
Why does this need to be optional?

Reply:
It's set to null by default in SubjectConfig just to match the other properties

Reply:
Do you recommend I make it non-optional?

Reply:
I think that would be slightly better. It's a bit confusing what does the `null` state stand for and in the `TrackerConfiguration` we tend to use non-optional booleans for configuration like this.